defaults/main.yml

```diff
@@ -69,9 +73,9 @@ if ansible_distribution in ['RedHat', 'CentOS', 'OracleLinux'] and
 # |select() filters the list to include only values that evaluate to true
 #     (the empty string is false)
 # |list() converts the generator that |select() creates to a list
-network_packages_default_initscripts: "{{ ['ethtool']
-+ _network_packages_default_initscripts_bridge|select()|list()
-+ _network_packages_default_initscripts_network_scripts|select()|list()
+__network_packages_default_initscripts: "{{ ['ethtool']
```

Comment:
If it is not too much trouble, it would be great if you put the commit to rename the variables in its own PR, so that we can merge it already. PSA: #231 will conflict with this change.

Reply:
Have created PR here #233